src/gui/SettingsWidgetGeneral.ui

```diff
@@ -367,6 +367,13 @@
      </widget>
     </widget>
    </item>
+   <item row="15" column="0">
+    <widget class="QCheckBox" name="autoCloseOnScreenLockCheckBox">
```

Comment:
Yup, I forgot to remove this after the cherry-pick as this was added by the previous (old) commit